cmd/add/social/social.go

```diff
@@ -47,39 +48,71 @@ func NewsocialCmd() *cobra.Command {
 		Use:     "social",
 		Short:   "add social provider",
 		Long:    `This commmand adds social provider`,
-		Example: `$ lr add social --provider <provider>`,
+		Example: `$ lr add social`,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if opts1.Provider == "" {
-				return &cmdutil.FlagError{Err: errors.New("`provider` is require argument")}
-			}
 
-			if opts1.ProviderKey == "" {
-				return &cmdutil.FlagError{Err: errors.New("`ProviderKey` is require argument")}
-			}
-
-			if opts1.ProviderSecret == "" {
-				return &cmdutil.FlagError{Err: errors.New("`ProviderSecret` is require argument")}
-			}
 			return add1(opts1)
 
 		},
 	}
 
-	fl := cmd.Flags()
-	fl.StringVarP(&opts1.Provider, "provider", "p", "", "provider name")
-	fl.StringVarP(&opts1.ProviderKey, "ProviderKey", "k", "", "ProviderKey")
-	fl.StringVarP(&opts1.ProviderSecret, "ProviderSecret", "s", "", "ProviderSecret")
 	return cmd
 }
 
 func add1(opts1 *socialProvider) error {
+	conf := config.GetInstance()
+
+	res, err := api.GetSites()
+	if err != nil {
+		return err
+	}
+	var num int
+	if res.Productplan.Name == "free" {
+		for i := 0; i < 3; i++ {
+			fmt.Println(i+1, arr[i])
+		}
+		fmt.Print("Please select a number from 1 to 3 :")
+		fmt.Scanln(&num)
+		for 1 > num || num > 3 {
+			fmt.Print("Please select a number from 1 to 3 :")
+
+			fmt.Scanln(&num)
+		}
+	}
+	if res.Productplan.Name == "developer" {
+		for i := 0; i < len(arr); i++ {
```

Comment:
What happened if it doesn't belong to any of the condition? for every `if` there should be `else` case.